core/src/wallet/ethereum/api_impl/EthereumLikeOperation.h

```diff
@@ -35,16 +35,18 @@
 #include <api/EthereumLikeOperation.hpp>
 #include <api/EthereumLikeTransaction.hpp>
 #include <wallet/common/api_impl/OperationApi.h>
+#include <api/InternalTransaction.hpp>
 
 namespace ledger {
     namespace core {
         class EthereumLikeOperation : public api::EthereumLikeOperation {
         public:
             EthereumLikeOperation(const std::shared_ptr<OperationApi>& baseOp);
-            std::shared_ptr<api::EthereumLikeTransaction> getTransaction();
-
+            std::shared_ptr<api::EthereumLikeTransaction> getTransaction() override;
+            std::vector<std::shared_ptr<api::InternalTransaction>> getInternalTransactions() override;
```

Comment:
This makes me think that we should have a way to annotate djinni functions with constness qualifiers, because it would be good to have it there!